src/panoptes/pocs/scheduler/observation/observation.py

```diff
@@ -40,25 +42,28 @@ def __init__(self, field, exptime=120 * u.second, min_nexp=60,
                 (default: {100})
             filter_name {str} -- Name of the filter to be used. If specified,
                 will override the default filter name (default: {None}).
-
+            dark (bool, optional): If True, exposures should be taken with the shutter closed.
+                Default: False.
         """
         super().__init__(*args, **kwargs)
 
-        assert isinstance(field, Field), self.logger.error("Must be a valid Field instance")
+        if not isinstance(field, Field):
+            raise TypeError(f"field must be a valid Field instance, got {type(field)}.")
 
-        assert exptime > 0.0, \
-            self.logger.error(f"Exposure time (exptime={exptime}) must be greater than 0")
+        if not exptime >= 0.0:  # 0 second exposures correspond to bias frames
+            raise ValueError(f"Exposure time must be greater than or equal to 0, got {exptime}.")
 
-        assert min_nexp % exp_set_size == 0, \
-            self.logger.error(
-                f"Minimum number of exposures (min_nexp={min_nexp}) must be " +
-                f"multiple of set size (exp_set_size={exp_set_size})")
+        if not min_nexp % exp_set_size == 0:
+            raise ValueError(f"Minimum number of exposures (min_nexp={min_nexp}) must be "
+                             f"a multiple of set size (exp_set_size={exp_set_size}).")
```

Comment:
We're actually back on 3.8 if you want to do:

```suggestion
            raise ValueError(f"Minimum number of exposures ({min_nexp=}) must be "
                             f"a multiple of set size ({exp_set_size=}).")
```